mappings/net/minecraft/client/render/model/BakedQuad.mapping

```diff
@@ -1,4 +1,5 @@
 CLASS net/minecraft/class_777 net/minecraft/client/render/model/BakedQuad
+	FIELD field_22441 directionalLighting Z
```

Comment:
Is this like the `"gui_light": "side"` in item model? I feel this name itself isn't very clear and might get some javadocs.

Reply:
That's a very good question. I tried to trace where it was coming from, and other parts refer to it as "shade/d" (which I will take responsibility for)

Neither name is idea, so i may look into it a little further to see if I can trace it back to an actual value in the json resources.

Reply:
The value traces back to `ModelElement$Deserializer` where the member name is called "shade" so I've changed everything to use that term.

```
            if (jsonObject5.has("shade") && !JsonHelper.hasBoolean(jsonObject5, "shade")) {
                throw new JsonParseException("Expected shade to be a Boolean");
            }
            boolean boolean10 = JsonHelper.getBoolean(jsonObject5, "shade", true);
```

Reply:
Perhaps it still needs a javadoc, because shade isn't exactly a clear meaning...